### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Deobp**

### CODE REVIEW

**Comment by Deobp** [comment]
reopen

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,62 @@
...
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="container">
+      <div class="stars">
+        <div class="modifier stars--001"></div>
```
Comment (mykhalenych) [inline src/index.html:14]: Screenshot 2023-09-04 at 17 05 40


Screenshot 2023-09-04 at 17 05 13

File: readme.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # Stars block
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_stars/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_stars/report/html_report/)
+- [DEMO LINK](https://Deobp.github.io/layout_stars/)
+- [TEST REPORT LINK](https://Deobp.github.io/layout_stars/report/html_report/)
```
Comment (loralevitska) [inline readme.md:4]: Add this links to PR description 
image
image

File: src/index.html [hunk 3]
```diff
@@ -1,13 +1,62 @@
...
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="container">
+      <div class="stars modifier stars--0">
```
Comment (loralevitska) [inline src/index.html:13]: Remove redundant unused class

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,41 @@
+* {
+  margin: 0;
+  padding: 0;
```
Comment (loralevitska) [inline src/style.css:3]: Only `box-sizing` property allows for `*` selector

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url("images/star.svg");
+  height: 16px;
+  width: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
```
Comment (loralevitska) [inline src/style.css:15]: Combine with `background` property

File: src/style.css [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+  background-image: url("images/star-active.svg");
+}
+
+.stars:nth-child(5) > *:nth-child(n+1):nth-child(-n+4) {
+  background-image: url("images/star-active.svg");
+}
+
+.stars:nth-child(6) > *:nth-child(n+1):nth-child(-n+5) {
+  background-image: url("images/star-active.svg");
+}
```
Comment (loralevitska) [inline src/style.css:41]:
```suggestion
.stars--1 :nth-child(-n + 1),
.stars--2 :nth-child(-n + 2),
.stars--3 :nth-child(-n + 3),
.stars--4 :nth-child(-n + 4),
.stars--5 :nth-child(-n + 5) {
  background-image: url("images/star-active.svg");
}
```

**Review by loralevitska — CHANGES_REQUESTED** [review]
Almost perfect) Let's improve

File: src/style.css (hunk 4 above)
Comment (Deobp) [reply src/style.css:3]: Why?

Comment (anastasiiavorobiova) [reply src/style.css:3]: Not fixed

File: src/style.css (hunk 6 above)
Comment (anastasiiavorobiova) [reply src/style.css:41]: Not fixed

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Great work! Unfortunately, changing a modifier doesn't work Screenshot 2023-09-04 at 19 01 27

**Pull request closed without merging**